backend/app/app.routes.js

```diff
@@ -1,25 +1,27 @@
 import express from "express";
+import auth from "./app.auth";
 import passport from "passport";
 
 const router = express.Router();
 
 router.get("/", (req, res) => {
-  res.render("index", {
-    isAuthenticated: !!req.session.currentUser,
-    error: req.query.error,
-  });
+  if (auth.isUserLoggedIn(req)) {
+    return res.redirect("/morpheus");
+  }
+
+  return res.render("index", { error: req.query.error });
 });
```

Comment:
Isn't it better to add a `home.controller.js` to handle this logic?

Something like this:

```javascript
import Home from './controllers/home.controller';

router.get("/", Home.index);
```

Reply:
I think we can extract all route functions  on a separated pr. What do you think?

Reply:
The index is the login page 😅 .
I don't like javascript classes for imports as much. My personal preference is to export only functions and explicit import what do you want.

What do you think to discuss about the backend structure / code on a issue?

Reply:
Perfect, I agree!!! 👏